tests/sqlness/src/setup.rs

```diff
@@ -0,0 +1,69 @@
+use std::{
+    env,
+    fs::File,
+    process::{Child, Command},
+};
+
+use async_trait::async_trait;
+use ceresdb_client_rs::db_client::{Builder, Mode};
+use sqlness::Environment as SqlnessEnv;
+
+use crate::client::Client;
+
+const BINARY_PATH_ENV: &str = "CERESDB_BINARY_PATH";
+const CONFIG_PATH_ENV: &str = "CERESDB_CONFIG_PATH";
+const SERVER_ENDPOINT_ENV: &str = "CERESDB_SERVER_ENDPOINT";
+const CERESDB_STDOUT_FILE: &str = "CERESDB_STDOUT_FILE";
+const CERESDB_STDERR_FILE: &str = "CERESDB_STDERR_FILE";
+
+pub struct Environment {
+    _server_process: Child,
+}
+
+#[async_trait]
+impl SqlnessEnv for Environment {
+    type DB = Client;
+
+    async fn start(&self, _mode: &str, _config: Option<String>) -> Self::DB {
+        let endpoint = env::var(SERVER_ENDPOINT_ENV).unwrap_or_else(|_| {
+            panic!(
+                "Cannot read server endpoint from env {:?}",
+                SERVER_ENDPOINT_ENV
+            )
+        });
+
+        let client = Builder::new(endpoint, Mode::Standalone).build();
+        Client::new(client)
+    }
+
+    /// Stop one [`Database`].
+    async fn stop(&self, _mode: &str, _database: Self::DB) {
+        // self.server_process.kill().unwrap()
```

Comment:
Maybe you have to rebase the main branch. After all, the #461 have fixed one bug for test failure.